pytest/tests/loadtest/locust/common/social.py

```diff
@@ -291,7 +291,7 @@ def on_locust_init(environment, **kwargs):
             )
 
         social_contract_code = environment.parsed_options.social_db_wasm
-        contract_key = key.Key.from_random(environment.social_account_id)
+        contract_key = key.Key.from_seed_testonly(environment.social_account_id)
```

Comment:
I believe you need two arguments for `Key.from_seed_testonly` [1]

[1] https://github.com/near/nearcore/blob/385e9f1f8ed1ec00cc5128fefb183c975b60ede0/pytest/lib/key.py#L41

Reply:
Good catch! I lost that one while untangling ~10 commits into 4 PRs, I hope it's the only mistake I made...

(Also, we seem to always use the same values for both args, maybe we should refactor to simplify?)

Reply:
Agreed. I can imagine a use case where one would require to customize a seed, but so far we've been fine with using the same default seed value. We can either add another method `from_default_seed` or just add a default value to the `seed` argument in this function.

Reply:
#9234